SETUP.md

```diff
@@ -1,30 +1,30 @@
 # Welcome!
 
-It's quick and easy to set up your environment to build [New Relic One applications](https://docs.newrelic.com/docs/new-relic-programmable-platform-introduction). In these setup instructions, you'll be setting up your system to run a local server that establishes a connection to the New Relic One platform. This connection makes building your New Relic One application easy because you can see your changes in real time. Enjoy!
+You can set up your environment to build [New Relic One applications](https://docs.newrelic.com/docs/new-relic-programmable-platform-introduction) in three steps. By the end of this guide, you'll have a local server with an established connection to the New Relic One platform, where you can see changes to your app in real time. Enjoy!
 
-# Setup Instructions
+## Setting up your environment
```

Comment:
Style guide: All titles and descriptions should be sentence case.

Also updated headings for parallel structure.